.gitignore

```diff
@@ -2,4 +2,6 @@
 *__pycache__
 client_secrets.json
 dashboard/static/js
+dashboard/templates/index.html
```

Comment:
@achen2401 are you sure about ignoring this `index.html` file?

Reply:
yes, the frontend code should auto-generate the file with required js files injected.  I made use of this webpack plugin `HtmlWebpackPlugin`, specified in webpack.config.js